src/expressions/struct-expr.md

```diff
@@ -101,7 +101,8 @@ Point3d { x, y: y_value, z };
 ## Tuple struct expression
 
 A struct expression with fields enclosed in parentheses constructs a tuple struct.
-Though it is listed here as a specific expression for completeness, it is equivalent to a [call expression] to the tuple struct's constructor. For example:
+Though it is listed here as a specific expression for completeness, it is equivalent to a [call expression] to the tuple struct's constructor. For example: <!--
+This is false. Example: Position(0, ..other_position) -->
```

Comment:
I don't understand this comment. Can you show what it means? 
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=a295289c9b9e281e95c80e220b3fd3ad

Reply:
Oh, I figured that would work. Guess it does not, and its falseness is false.